recipes/rstudio/meta.yaml

```diff
@@ -49,7 +49,6 @@ source:
       - patches/0039-Add-support-for-Qt-5.10.patch
       - patches/0040-Revert-disable-macOS-specific-Cmd-Shift-handling-clo.patch
       - patches/0041-Fix-missing-boost-placeholders.patch
-      - patches/0042-Conda-We-use-a-shared-soci-library.patch
```

Comment:
There was nothing about `SOCI_LIBRARIES` in 1.3.1073, particularly not in `src/cpp/CMakeLists.txt`. Skipped this patch.